services/bettersocial/fcmToken/sendCommentNotification.js

```diff
@@ -17,11 +17,12 @@ const sendCommentNotification = async (userTargetId, commentAuthor, message, act
     data: {
       feed_id: activity_id,
       type: 'feed'
-    }
+    },
+    token: userTargetToken?.token
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `sendToDevice` to `send` is a good one, as it simplifies the code and makes it more readable. However, there's a potential issue with the new token assignment. If `userTargetToken` is undefined or null, `token: userTargetToken?.token` will be undefined. This could cause an error when trying to send the payload.

```diff
-    token: userTargetToken?.token
+    token: userTargetToken ? userTargetToken.token : ''
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->